aten/src/ATen/Storage.h

```diff
@@ -21,6 +21,9 @@ struct AT_API Storage {
   Storage(const Storage&) = delete;
   Storage(Storage&&) = delete;
   Storage(const Storage&&) = delete;
+  void set_pImpl(StorageImpl* storage_impl) {
+    storage_impl_ = storage_impl;
```

Comment:
This looks like a dangerous function. Is it really needed?

Reply:
OK, looks like it's just used by THLongStorageView, which is going away. If you want to be nice add a comment saying when we can delete this function, but since we plan to get rid of this soon it's not totally necessary.